fluent-resmgr/Cargo.toml

```diff
@@ -22,6 +22,7 @@ fluent-fallback = { version = "0.6.0", path = "../fluent-fallback" }
 unic-langid = "0.9"
 elsa = "1.5"
 futures = "0.3"
+thiserror = "1.0.37"
```

Comment:
Suggestion: The version number should match the other `thiserror` version.

Reply:
Is this ok if we start using workspace dependencies to simplify the versions across the crates? See this commit https://github.com/projectfluent/fluent-rs/pull/282/commits/a75bf0858e21b88a61292122b14efb34caae07ba

Reply:
This might be better as a separate PR. I don't know if the [Firefox vendoring process](https://firefox-source-docs.mozilla.org/build/buildsystem/rust.html) works with workspace, and we would need to block on verifying that it would work. I wouldn't want to break the Firefox integration.